app/components/searchBar/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useState } from 'react';
+
+import { MagnifyingGlassIcon } from '@heroicons/react/24/outline';
+
+//for passing props creating interface
+interface SearchProps {
+  placeholder?: string;
+  defaultValue?: string;
+  onChangeValue: (value: string) => void;
+}
+
+const SearchBar: React.FC<SearchProps> = ({
+  placeholder = 'Search',
+  defaultValue,
+  onChangeValue,
+}) => {
+  return (
+    <div className="flex justify-center bg-gray-100 rounded-xl  border-2 overflow-hidden w-80 m-2">
+      <MagnifyingGlassIcon className="h-8.5 w-5 mx-2 text-gray-500" />
+      <input
+        data-testid="searchBar"
+        type="search"
+        placeholder={placeholder}
+        defaultValue={defaultValue}
+        onChange={(e) => onChangeValue(e.target.value)}
+        className="block rounded-md border-0 flex-grow p-2 bg-gray-100 !outline-none"
```

Comment:
because when you click it will not show any border 
![Screenshot from 2023-02-24 21-17-32](https://user-images.githubusercontent.com/71075101/221223150-167c654e-c556-418a-ba04-354237afeffb.png)

OtherWise:
![Screenshot from 2023-02-24 21-17-22](https://user-images.githubusercontent.com/71075101/221223339-c30512a7-7bc6-453d-b862-39973e1f3719.png)

Reply:
added because: 
![Screenshot from 2023-02-24 21-17-32](https://user-images.githubusercontent.com/71075101/221224228-d4e39085-f968-488d-b8cb-df77b55f6e5e.p

otherwise: 
![Screenshot from 2023-02-24 21-17-22](https://user-images.githubusercontent.com/71075101/221224308-14e14499-1bbe-44f9-8572-39c80c06d077.png)
ng)